orb-ui/src/engine/pearl.rs

```diff
@@ -175,24 +175,24 @@ impl Runner<PEARL_RING_LED_COUNT, PEARL_CENTER_LED_COUNT> {
         self.center_animations_stack.set(level, Box::new(animation));
     }
 
-    fn stop_ring(&mut self, level: u8, force: bool) {
-        self.ring_animations_stack.stop(level, force);
+    fn stop_ring(&mut self, level: u8, transition: Transition) {
+        self.ring_animations_stack.stop(level, transition);
     }
 
-    fn stop_center(&mut self, level: u8, force: bool) {
-        self.center_animations_stack.stop(level, force);
+    fn stop_center(&mut self, level: u8, transition: Transition) {
+        self.center_animations_stack.stop(level, transition);
     }
 }
 
 #[async_trait]
 impl EventHandler for Runner<PEARL_RING_LED_COUNT, PEARL_CENTER_LED_COUNT> {
     #[allow(clippy::too_many_lines)]
     fn event(&mut self, event: &Event) -> Result<()> {
-        tracing::trace!("UI event: {}", serde_json::to_string(event)?.as_str());
+        tracing::debug!("UI event: {}", serde_json::to_string(event)?.as_str());
```

Comment:
Imo high frequency messages should be trace level

Reply:
I want to keep this log line as debug because it helps a lot with debugging and this line is actually pretty much the only one being printed when orb-ui runs with debug logs enabled.